lib/engine/game/g_1856/step/buy_train.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/buy_train'
+
+module Engine
+  module Game
+    module G1856
+      module Step
+        class BuyTrain < Engine::Step::BuyTrain
+          def must_buy_train?(entity)
+            return super unless entity == @game.national && !@game.national_ever_owned_permanent
+
+            cash = entity.cash
+            # Can't use empty because the national may have a permanent train borrowed :(
+            has_tradeable_train = entity.trains.any?(&:rusts_on)
+            cash > 1100 || (has_tradeable_train && cash > 750)
```

Comment:
Not a fan of the magic numbers here.

Reply:
Changed

Reply:
Might not have pushed

Reply:
I've pushed now, I was waiting to get the spec in and working first